template_tests/filters.tpl.out

```diff
@@ -22,13 +22,13 @@ cut
 Helloworld
 
 default
-n/a
+<map[string]interface {} Value>
 42
 42
 5
 
 default_if_none
-n/a
+<map[string]interface {} Value>
```

Comment:
Are you sure if this output should be the one expected. Are we updating this to pass tests? If so, we should check if tests started failing because of our changes or they were already failing when we forked the repo. If former, I would prefer tests failing than for tests to pass incorrectly. Writing based on limited understanding.

Reply:
please see my previous comment, test output has been modified due to our changes, I am just highlighting the difference here. Who can fix them? we need to do it asap so that we can enable test check in prs

Reply:
I think we should merge this PR for now and fix the issue at hand. 
I will get these 2 things done separately. 
- ci test run setup for this repo
- fixing the value issues